lib/Models/WebMapServiceCatalogItem.js

```diff
@@ -515,14 +644,40 @@ WebMapServiceCatalogItem.prototype.updateFromCapabilities = function(capabilitie
         myLayer = myLayer[0];
         if (myLayer && myLayer.Style && myLayer.Style.length > 0) {
           for (var j = 0; j < myLayer.Style.length; ++j) {
-            if (!defined(this.isNcWMS) && myLayer.Style[j].Name && myLayer.Style[j].Name.match(/boxfill\/rainbow/i)) {
+            if (!defined(this.isNcWMS) && myLayer.Style[j].Name &&
+                (myLayer.Style[j].Name.match(/boxfill\/rainbow/i) ||
+                 myLayer.Style[j].Name.match(/default-scalar\/default/i) ||
+                 myLayer.Style[j].Name.match(/default-vector\/default/i))) {
```

Comment:
No, it's just a heuristic.  The ncWMS developer recently added a way to reliably detect the availability of COLORSCALERANGE (see #1976), but we can't rely on that yet exclusively yet because it's so new.  So basically we guess based on the presence of these styles, and the user can override our guess by explicitly setting the property values.  #1976 has more discussion.